pulsar/consumer.go

```diff
@@ -91,7 +93,7 @@ type ConsumerOptions struct {
 
 	// InitialPosition at which the cursor will be set when subscribe
 	// Default is `Latest`
-	SubscriptionInitPos InitialPosition
+	SubscriptionInitPos SubscriptionInitialPosition
```

Comment:
We could leave the variable name out here since it's the same as the enum type.

Reply:
I was trying to be more descriptive for example:
```
pulsar.InitialPosition
vs
pulsar.SubscriptionInitialPosition
```
The second one seemed clearer to me. Thoughts?